docs/rdiff-backup.1.adoc

```diff
@@ -333,11 +331,11 @@ The default is to include other filesystems.
 
 --min-file-size _sizeinbytes_:: Exclude files that are smaller than the given size in bytes.
 
-[[statistics-options]]
+[[satistics-options]]
```

Comment:
Sorry,  I see a missing t here.

Reply:
Done

Reply:
Something broke in the testing, it can't find setuptools-scm.

Reply:
There were CDN issues during the day,  just amend the last commit and force push it.

Reply:
Sorry, there is a re run jobs button in the checks tab, no need to force push!